tests/test_pyctuator_e2e.py

```diff
@@ -215,13 +215,13 @@ def test_logfile_endpoint(endpoints: Endpoints) -> None:
     thirsty_str = "These pretzels are making me thirsty"
     response: Response = requests.get(
         endpoints.root + "logfile_test_repeater",
-        params={"repeated_string": "thirsty_str"}
+        params={"repeated_string": f"{thirsty_str}"}
```

Comment:
nice catch, though i dont think you need the `f"...` at all - you can put `thirsty_str` directly.